### DISCUSSION TITLE
See my other question.

### DISCUSSION THREAD

**Discussion by Echo69420** [community/community#35855] [Other Feature Feedback, Questions, & Ideas]
I really need help with a skript thing. Idk how often people post here but it should be on the Skript thread and I would assume my question would also show up if you click on my name (it's my only post besides this) Please help. Thank you

**Comment by MitkoVtori** [comment]
I am not fully sure what do you been by this, but here people ask questions, share information and are being one cool community that's ready to give feedback at anytime.

**Reply by Echo69420** [reply]
Wait is this under skript

**Reply by MitkoVtori** [reply]
Maybe, what do you mean by under skript? Do you mean like JavaScript, **sCript**?

**Reply by Echo69420** [reply]
Ok so it’s not under Skript lol

**Reply by NhTushar70** [reply]
Nice
